<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTag_BasicParams</class>
 <widget class="QDialog" name="CreateTag_BasicParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox_sun">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>720</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sunday</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Description (optional)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;A brief description of what the Tag is for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_enabledDays">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>610</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Enabled Days (advanced)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Choose which days of the week should this Tag be active on.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;If you don't have specific restrictions, keep this Tag enabled on all days.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_wed">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>640</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Wednesday</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_sat">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>700</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturday</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;A name for your Tag, such as: Vegetarian, Dessert, Weekends Only, etc.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_thu">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>660</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Thursday</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_tue">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Tuesday</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>enter a name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_multitagDependency">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;MultiTag Dependency (advanced)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Disable this Tag unless linked with a MultiTag?&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;(Recommended: no)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Consecutive Days Limit&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Limit how many times &lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;all Meals planned by this Tag&lt;/span&gt; may occur back-to-back. &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;This setting applies only to this Tag.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;A Meal assigned to multiple Tags will therefore follow multiple rules.&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;1 means Meals planned under this Tag may occur at most every other day.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;3 means Meals under this Tag may be planned for at most 3 days in a row.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_consecutiveDaysLimit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dependsOnMultiTag">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_fri">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>680</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Friday</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_description">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>80</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>(example) Contains Meals suitable for picnics.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_mon">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>600</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Monday</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>720</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>720</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
